src/Simplex/Messaging/Agent/Store/SQLite/Migrations.hs

```diff
@@ -128,9 +128,13 @@ run st = \case
   where
     runUp Migration {name, up, down} = withTransaction' st $ \db -> do
       when (name == "m20220811_onion_hosts") $ updateServers db
-      insert db >> execSQL db up
+      insert db >> execSQL db up'
       where
         insert db = DB.execute db "INSERT INTO migrations (name, down, ts) VALUES (?,?,?)" . (name,down,) =<< getCurrentTime
+        -- we could add an optional field for new database migration to the list
```

Comment:
I don't think it's a good idea to legitimize this approach, this is an exceptional case. Each such "revised" migration would split user base into separate categories in terms of possible errors
```suggestion
```